kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -587,7 +587,7 @@ public static ConfigDef newConfigDef() {
           BehaviorOnNullValues.VALIDATOR,
           Importance.LOW,
           "How to handle records with a null value (i.e. Kafka tombstone records)."
-              + " Valid options are 'ignore' and 'fail'.",
+              + " Valid options are 'ignore', 'fail', and 'keep'",
```

Comment:
A downside of adding this option is that, AFAIK, it is not compatible to any of the built in existing formats/writers. Someone reading the notes here may very well be mislead into thinking that the `keep` option will work with the "in-the-box" options... would adding notes/documentation here about this limitation the appropriate place?  Is there an alternative approach to the general problem of this PR that'd be advised?